src/modules/ourComments.js

```diff
@@ -0,0 +1,88 @@
+const addComment = (id, name, msg) => {
+  if (name.value !== '' && msg.value !== '') {
+    // if name and msg not empty add New
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Hi, Let us not disable linter, because a linter is a tool that analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs. It is prohibited to disable it.

Reply:
fixed

Reply:
fixed